samples/brainfuck.cr

```diff
@@ -43,6 +47,7 @@ class Program
       when '+'; tape.inc
       when '-'; tape.dec
       when '.'; print tape.get.chr
+      when ','; if byte = STDIN.read_byte; tape.set byte.to_i; end
```

Comment:
Block version reads IMO a bit better as a oneliner:

```suggestion
      when ','; STDIN.read_byte.try { |byte| tape.set byte.to_i }
```

Reply:
I agree, your version fits the style better.